docs/production.md

```diff
@@ -0,0 +1,69 @@
+
+If you encounter any errors during the merge, the production branch status will be updated to `MIGRATIONS_FAILED`. You can ask the LLM to lookup the exact error for this branch action using the `get_logs` tool. To fix these errors, you must follow these steps.
+
+1. Reset the problematic migration from your development branch.
+2. Apply a new migration with the fix on your development branch.
+3. Merge the development branch to production.
+
+Only successful migrations are tracked so it is safe to merge the same development branch multiple times.
+
+### Delete a development branch
+
+Finally, after merging all changes to production, you can delete the development branch using the `delete_branch` tool. This helps you save on resources as any active development branch will be billed at $0.01344 per hour.
+
+### Rebase a development branch
+
+Sometimes it is unavoidable to apply a hotfix migration on your production database directly. As a result, your development branch may be behind your production branch in terms of migration versions.
+
+Similarly, if you are working in a team where each member works on a separate development branch, merging branches in different order could also result in migration drift.
+
+To fix this problem, you can either recreate your development branch or invoke the `rebase_branch` tool. This tool incrementally applies new migrations from the production branch back on to the development branch.
+
+### Conclusion
+
+To summarise our workflow using development and production branches, we expose 3 core tools for managing migrations.
+
+1. `rebase_branch`: This tool brings the development branch in sync with the production branch, covering cases where production is ahead of development. Creating a new development branch runs this tool implicitly. If you use multiple development branches, merging branch A after creating branch B could also result in migration drift. You can run rebase on branch B to recover from drift.
+
+2. `merge_branch`: This tool brings production in sync with development, covering cases where development is ahead of production. Running this tool will apply new migrations from development to the production branch. Any failures should be resolved on the development branch before retrying.
+
+3. `reset_branch`: This tool is an escape hatch to cover all other cases where migrations are different between production and development. By default it resets the development branch to the latest migration, dropping any untracked tables and data. You can also specify a prior migration version to revert a migration that's already applied on development. A version of 0 will reset the development to a fresh database.
+
+Mastering this workflow goes a long way to ensure your production app is always ready when you release new features and bug fixes. Happy vibe coding!
```

Comment:
Love the ending but I think we should omit "vibe coding" mainly because it can have different connotations.

```suggestion
Mastering this workflow goes a long way to ensure your production app is always ready when you release new features and bug fixes.
```